don't use bundle getopt # bug 945512
--- a/Makefile.in
+++ b/Makefile.in
@@ -115,8 +115,6 @@ man_MANS = checkpassword-pam.8
 bin_PROGRAMS = checkpassword-pam
 
 checkpassword_pam_SOURCES = checkpassword-pam.c logging.h pam-support.h pam-support.c \
-                            getopt.h getopt.c getopt1.c
-
 
 EXTRA_DIST = $(man_MANS) interface.html \
              debian/changelog debian/control debian/copyright \
@@ -131,7 +129,7 @@ bin_PROGRAMS = checkpassword-pam$(EXEEXT)
 PROGRAMS = $(bin_PROGRAMS)
 
 am_checkpassword_pam_OBJECTS = checkpassword-pam.$(OBJEXT) \
-	pam-support.$(OBJEXT) getopt.$(OBJEXT) getopt1.$(OBJEXT)
+	pam-support.$(OBJEXT)
 checkpassword_pam_OBJECTS = $(am_checkpassword_pam_OBJECTS)
 checkpassword_pam_LDADD = $(LDADD)
 checkpassword_pam_DEPENDENCIES =
@@ -141,7 +139,6 @@ DEFAULT_INCLUDES =  -I. -I$(srcdir) -I.
 depcomp = $(SHELL) $(top_srcdir)/depcomp
 am__depfiles_maybe = depfiles
 @AMDEP_TRUE@DEP_FILES = ./$(DEPDIR)/checkpassword-pam.Po \
-@AMDEP_TRUE@	./$(DEPDIR)/getopt.Po ./$(DEPDIR)/getopt1.Po \
 @AMDEP_TRUE@	./$(DEPDIR)/pam-support.Po
 COMPILE = $(CC) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(AM_CPPFLAGS) \
 	$(CPPFLAGS) $(AM_CFLAGS) $(CFLAGS)
@@ -232,8 +229,6 @@ distclean-compile:
 	-rm -f *.tab.c
 
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/checkpassword-pam.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/getopt.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/getopt1.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/pam-support.Po@am__quote@
 
 .c.o:
